### DISCUSSION TITLE
如何实现变身WiFi，理论可以实现。有无大佬可以合作实现。

### DISCUSSION THREAD

**Discussion by yizhiyian** [community/community#82189] [Programming Help]
### Body

变身WiFi，通过屏蔽对方的WiFi，克隆对方的WiFi，使得用户自动连接伪造WiFi，对其上传下载数据的数据的拦截，并自动备份到服务器。类似于钓鱼，但是前提是为屏蔽对方WiFi，使得对方WiFi，无法被检测，从而自动连接我的虚假钓鱼WiFi。欢迎大佬给予建议和指导。

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Akash1134** [comment]
Hi @yizhiyian , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
